main/zoe/api/zoe-contract-facet.md

```diff
@@ -161,7 +162,9 @@ await zcf.saveIssuer(secondaryIssuer, keyword);
 
 Make a credible Zoe `invitation` for a smart contract. Note that `invitations` are a special case
 of an ERTP `payment`. They are in a one-to-one relationship with the `invitationIssuer`, which is used
-to validate invitations and their `amounts`.
+to validate invitations and their `amounts`. They are also in a one-to-one relationship with
+a `mint` devoted to minting new `invitations`. `zcf.makeInvitation()` serves as an interface to
```

Comment:
I like the comment about the interface to the mint. See above for the one-to-one comment